enterprise/backend/enterprise_core/settings.py

```
@@ -128,8 +128,44 @@ def set_ciso_assistant_url(_, __, event_dict):
+        "AWS_STORAGE_BUCKET_NAME", "ciso-assistant-bucket"
+    )
+    AWS_S3_ENDPOINT_URL = os.getenv("AWS_S3_ENDPOINT_URL")
+
+    if not AWS_ACCESS_KEY_ID:
+        logger.error("AWS_ACCESS_KEY_ID must be set")
+    if not AWS_SECRET_ACCESS_KEY:
+        logger.error("AWS_SECRET_ACCESS_KEY must be set")
+    if not AWS_S3_ENDPOINT_URL:
+        logger.error("AWS_S3_ENDPOINT_URL must be set")
+    if not AWS_ACCESS_KEY_ID or not AWS_SECRET_ACCESS_KEY or not AWS_S3_ENDPOINT_URL:
+        raise ImproperlyConfigured("AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, and AWS_S3_ENDPOINT_URL must be set when USE_S3=True")
```

Ruff (0.8.2)

Undefined name `ImproperlyConfigured`

(F821)